pulp_rpm/app/models/repository.py

```diff
@@ -151,6 +155,8 @@ class RpmPublication(Publication):
     """
 
     TYPE = 'rpm'
+    # metadata_checksum_type = models.CharField(max_length=20)
+    # package_checksum_type = models.CharField(max_length=20)
```

Comment:
@goosemania @ipanova do you think I have to save these data?

Reply:
It make sense to me, it's a part of publication configuration